TramsDataApi.Test/Integration/TrustsIntegrationTests.cs

```diff
@@ -379,7 +380,55 @@ public async Task ShouldReturnSubsetOfTrusts_WhenSearchingTrusts_ByGroupName()
            
             _legacyDbContext.SaveChanges();
         }
-        
+
+        [Fact]
+        public async Task ShouldReturnSubsetOfTrusts_WhenSearchingTrusts_WithFuzzySearch_ByGroupName()
+        {
+
+            var groupNamePartial = "Mygroupnam";
```

Comment:
Might be worth having the search term have different upper and lower cases to ensure that the search is not case sensitive.